docs/tvl.md

```diff
@@ -13,19 +13,32 @@ As you can see there are three types of value:
 Everything you see here is achieved via proper configuration files in our repository.
 
 ## Token guide
-
-1. Go to `config\src\tokens\source.jsonc`
+1. Go to `config\src\tokens\tokens.jsonc`
 2. Add your token to the proper chain
 
-   - required fields are: **symbol & address**
-   - optional fileds are: type, formula, category, bridgedUsing & coingeckoId
-   - for `ethereum` the `type` and `formula` are `CBV` and `locked` by default, for other chains you need to specify it
-   - `category` is `other` by default
-   - if you do not see your chain run `yarn coingecko:platforms` (TODO: update docs when dynamic chains functionality is merged)
+→ Canonically bridged tokens (CBV) should always be added to Ethereum.
+- The required fields are `symbol` **&** `address`.
+- Optional fields are: `type`, `formula`, `category`, `bridgedUsing` & `coingeckoId`.
+- By default, the `type` is `CBV`, the `formula` is `locked`, and the `category` is `other`.
 
+→ Externally bridged tokens (EBV) and natively minted tokens (NMV) should be added to the token list of a specific L2.
```

Comment:
`→` aren't proper markdown syntax and break the flow of the list. Consider replacing them with `>` indented by three spaces.

Reply:
I decided to use three spaces only without the > since it seems like it creates a blockquote, which I think is not suitable, hopefully this works